pkg/cli/serve/serve.go

```diff
@@ -64,11 +67,12 @@ var (
 )
 
 const (
-	port                = 8081
-	accessTokenFlagName = "access-token"
-	servicesFlagName    = "services"
-	ocmURLFlagName      = "ocm-url"
-	debugFlagName       = "debug"
+	servicePort         int    = 8081
+	metricsPort         int    = 8888
```

Comment:
Although ultimately we can make the service port be different to that in the container, the operator sdk convention for the metrics port is `8383` (well, until 1.0, then it changed to 8080) so it might be worth using that for consistency.